LambdaEngine/Include/Rendering/RenderGraphEditor.h

```diff
@@ -77,6 +77,9 @@ namespace LambdaEngine
 		void SetInitialNodePositions();
 		void ResetState();
 
+		// Helpers
+		void RenderDimChangeOptions(EPipelineStateType pipelineStateType, int32 selectedOptions[3], float32 variable[3], float inputWidth);
```

Comment:
Naming is a bit confusing. "variable" should be more meaningful (see other comment)